json_serializable/lib/src/type_helpers/to_from_string.dart

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2019, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:analyzer/dart/element/type.dart';
+import 'package:source_gen/source_gen.dart';
+
+import '../type_helper.dart';
+
+const bigIntString = ToFromString(
+  'BigInt.parse',
+  'toString()',
+  TypeChecker.fromUrl('dart:core#BigInt'),
+);
+
+const dateTimeString = ToFromString(
+  'DateTime.parse',
+  'toIso8601String()',
+  TypeChecker.fromUrl('dart:core#DateTime'),
+);
+
+const uriString = ToFromString(
+  'Uri.parse',
+  'toString()',
+  TypeChecker.fromUrl('dart:core#Uri'),
+);
+
+/// Helper that unifies implementations of [Type]s that convert trivially
+/// to-from [String].
+class ToFromString {
```

Comment:
This name was really confusing to me at first, although I can't come up with a better one off the top of my head... maybe `StringSerializer` but that isn't really accurate either.

Reply:
Made it `ToFromStringHelper` 🤷‍♂ – this is very package-internal